test/apps/app/package.json

```diff
@@ -38,6 +38,9 @@
   "peerDependencies": {
     "@okta/okta-signin-widget": "^6.0.0"
   },
+  "resolutions": {
+    "follow-redirects": "^1.14.9"
```

Comment:
which dependency is this related to?

Reply:
`http-proxy-middleware`

https://github.com/okta/okta-auth-js/pull/1218/files#diff-26790002770945af943b6adbd98e2dcde649575b87198392113c3301ba5fcd8cR26